networking/ingress-operator.adoc

```diff
@@ -17,7 +17,16 @@ include::modules/nw-installation-ingress-config-asset.adoc[leveloffset=+1]
 
 include::modules/nw-ingress-controller-configuration-parameters.adoc[leveloffset=+1]
 
-include::modules/nw-ingress-controller-tls-profiles.adoc[leveloffset=+2]
+== Ingress Controller TLS security profile
+
+TLS security profiles provide a way for servers to regulate which ciphers a connecting client can use when connecting to the server.
+
+// Understanding TLS security profiles
+include::modules/tls-profiles-understanding.adoc[leveloffset=+3]
+
+// Configuring the TLS profile for the Ingress Controller
+include::modules/tls-profiles-ingress-configuring.adoc[leveloffset=+3]
+
 
 include::modules/nw-ingress-controller-endpoint-publishing-strategies.adoc[leveloffset=+2]
```

Comment:
Before, this module was underneath the "Ingress controller configuration parameters", and now it's under the TLS security profile heading, which I don't think is correct. I'd either move that module up under that, like:

```
Ingress controller configuration parameters
    Ingress controller endpoint publishing strategy
Ingress Controller TLS security profile
    Understanding TLS security profiles
    Configuring TLS security profiles for the Ingress controller
```

Or adjust the TLS profiles heading to be one more level in, like:

```
Ingress controller configuration parameters
    Ingress Controller TLS security profile
        Understanding TLS security profiles
        Configuring TLS security profiles for the Ingress controller
    Ingress controller endpoint publishing strategy
```
Though technically those two TLS modules might not show in the TOC anymore. But that's probably the right way to do it.

Reply:
That pushes the Understanding TLS security profiles and Configuring TLS security profiles for the Ingress controller to +3 level and are not in the TOC. I didn't like it at first. But now, i think it's fine. 
What about moving the modules under Configuring the Ingress Controller? Sections about other ingress controller parameters are there. 
@bmcelvee Any thoughts?